src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.EolTargetFrameworks.targets

```diff
@@ -20,7 +20,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     receive servicing updates and security fixes.
   -->
   <ItemGroup>
-    <_EolNetCoreTargetFrameworkVersions Include="1.0;1.1;2.0;2.1;2.2;3.0" />
+    <_EolNetCoreTargetFrameworkVersions Include="1.0;1.1;2.0;2.1;2.2;3.0;5.0" />
```

Comment:
3.1's life will end in [less than 90 days](https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core). should we add it here?

Reply:
Customers consider new warnings in existing products to be highly impactful.  We included a 2.1 warning the month after it went out of support and got a lot of feedback. After discussion internally when 5.0 went out of support, we decided to delay turning on this warning for 6 months after 5.0 went out of support to align with our 7.0 release to give customers more time before we rolled out the warning. I expect 3.1 will also be delayed by a few months before we add that warning.